neutrino.go

```diff
@@ -66,6 +67,10 @@ var (
 	// DisableDNSSeed disables getting initial addresses for Bitcoin nodes
 	// from DNS.
 	DisableDNSSeed = false
+
+	// DefaultFilterCacheSize is the size (in bytes) of filters neutrino will
+	// keep in memory if no size is specified in the neutrino.Config.
+	DefaultFilterCacheSize uint64 = 4096 * 1000
```

Comment:
Would be nice to include some back of the envelope calculation of how many filters will fit.

Reply:
Sure, sounds like a good idea, though I don't have any good numbers on the average filter sizes, etc. @Roasbeef do you have some stats I could use ?

Reply:
Not blocking for this PR, as I can add on later, but last I checked, the median filter size was ~14KB.